Profile Folder/chrome/locale/zh-CN/properties/gsettings.properties

```diff
@@ -1,12 +1,12 @@
-geckiumSettings=[!!! Gèçƙïú₥ §èƭƭïñϱƨ !!!]
+geckiumSettings=Geckium 设置
 
-addAnApp=Add an app
+addAnApp=添加应用程序
 
 themeHasNoDescription=This theme has no description.
-builtInTheme=[!!! [ßÛÌ£T-ÌN] {{themeName}} !!!]
+builtInTheme=[内置] {{themeName}}
 showInFolder=[!!! §λôω ïñ ƒôℓδèř !!!]
```

Comment:
`showInFolder` is the button that will take you to a theme's folder and selects it in the file manager.
![image](https://github.com/user-attachments/assets/7bb2a894-9a76-46c4-88a1-8d0a718d1263)